app/models/manageiq/providers/ansible_tower/shared/automation_manager/event_parser.rb

```diff
@@ -1,9 +1,8 @@
 module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::EventParser
   def event_to_hash(event, ems_id)
     {
-      :event_type => "#{self.event_type}_#{event.operation}",
+      :event_type => "#{event.object1}_#{event.operation}",
       :source     => "#{self.source}",
-      :message    => event.changes.to_s,
```

Comment:
```event.changes``` is a summary of the new and changed values when an object is created, updated, or deleted. Since ```event``` is stored in ```full_data``` there is no need to repeat it again in ```:message```.